server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -162,9 +166,13 @@ pub struct EndpointIn {
     #[validate(custom = "validate_url")]
     #[schemars(url, length(min = 1, max = 65_536), example = "example_endpoint_url")]
     pub url: Url,
-    #[validate(range(min = 1, message = "Endpoint versions must be at least one"))]
+
+    #[deprecated]
+    #[serde(default = "default_endpoint_version")]
+    #[validate(range(min = 1, message = "Endpoint versions must be at least one if set"))]
     #[schemars(range(min = 1), example = "example_endpoint_version")]
-    pub version: u16,
+    pub version: Option<u16>,
```

Comment:
This makes the field nullable, I think we just want optional (so just set default, don't allow null). I think this is fine in general for this change, but it's a subtle difference worth being aware of.